integration/e2e/overrides_client.go

```diff
@@ -0,0 +1,105 @@
+package e2e
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/grafana/tempo/modules/overrides"
+	"github.com/grafana/tempo/pkg/api"
+	tempoUtil "github.com/grafana/tempo/pkg/util"
+)
+
+// Client wraps tempoUtil.Client and adds functions to work with the user-configurable overrides API.
+// We can't add this functions to tempoUtil directly because this would result in an import cycle.
+type Client struct {
+	*tempoUtil.Client
+}
+
+func (c *Client) GetOverrides() (*overrides.UserConfigurableLimits, error) {
```

Comment:
current API client is in `/pkg/util/client.go`. i'm thinking we should move both of these to a new package. Something like `/pkg/util/httpclient`.

Reply:
edited the suggested path to `/pkg/httpclient`. overall i'm trying to move things out of util

Reply:
PR to move it: https://github.com/grafana/tempo/pull/2637
Once merged I can update this PR